build.gradle

```diff
@@ -62,9 +62,9 @@ import org.gradle.internal.logging.text.StyledTextOutputFactory
 import static org.gradle.internal.logging.text.StyledTextOutput.Style
 
 // Check for Java 8
-if(JavaVersion.current() != JavaVersion.VERSION_1_8){
+if(JavaVersion.current() != JavaVersion.VERSION_11){
```

Comment:
Aren't Java 1.8 and Java 9 `!= JavaVersion.VERSION_11`? 🧐 Did you test that with Java 8 and 9?

Reply:
it's probably fine — maybe even better — to support 8 **or** 11, instead of 8–11.

9 and 10 weren't long-term releases. 10 hasn't been maintained for a year and a half by this point.

Reply:
Whoops, thought that was a `>` not a `!=` good catch :-)

Will fix and test with a few more versions, been aiming to try it on two other systems. Can adjust logging for 8 or 11. Would we want to outright warn people if they're on 9 or 10?